xm.js

```diff
@@ -364,8 +365,8 @@ function MixChannelIntoBuf(ch, start, end, dataL, dataR) {
   var volE = ch.volE / 64.0;    // current volume envelope
   var panE = 4*(ch.panE - 32);  // current panning envelope
   var p = panE + ch.pan - 128;  // final pan
-  var volL = volE * (128 - p) * ch.vol / 8192.0;
-  var volR = volE * (128 + p) * ch.vol / 8192.0;
+  var volL = volE * (player.xm.global_volume - p) * ch.vol / 8192.0;
+  var volR = volE * (player.xm.global_volume + p) * ch.vol / 8192.0;
```

Comment:
yeah I checked the source code for OpenMPT which used `1 << 20` as a divisor (1048576) and the same formula. I'll fix and rebase :)